test/auth-password-store-tests.el

```diff
@@ -78,6 +78,20 @@ test code without touching the file system."
              (auth-pass--debug-log nil))
          ,@body))))
 
+(auth-pass-deftest any-host ()
+                   '(("foo" ("port" . "foo-port") ("host" . "foo-user"))
```

Comment:
indentation is not good. Can you please load the file and reindent? As soon as Emacs knows about `auth-pass-deftest`, indentation should get fixed.